launcher/src/Instance.scala

```diff
@@ -107,6 +111,13 @@ object Instance{
             case _ => ???
           }
         case multiple =>
+          Util.sentryCapture(
+            new EventBuilder()
+              .withMessage("Multiple devbox instances running")
+              .withLevel(Level.WARNING)
+              .withTag("whoami", System.getProperty("user.name"))
```

Comment:
Let's add some details about the instances themselves, perhaps creation date and state? We might be able to guess some patterns about the conditions under which we get to this state.